toolkit/scripts/pkggen.mk

```diff
@@ -89,13 +89,40 @@ $(specs_file): $(chroot_worker) $(BUILD_SPECS_DIR) $(build_specs) $(build_spec_d
 		$(if $(TARGET_ARCH),--target-arch="$(TARGET_ARCH)") \
 	        --output $@
 
+grapher_cycle_flags :=
+
+grapher_extra_flags :=
+ifeq ($(DISABLE_UPSTREAM_REPOS),y)
+grapher_extra_flags += --disable-upstream-repos
+endif
+
+ifeq ($(USE_PREVIEW_REPO),y)
+grapher_extra_flags += --use-preview-repo
+endif
+
+ifeq ($(RESOLVE_CYCLES_FROM_REPO),y)
+grapher_cycle_flags += --resolve-cycles-from-repo
+endif
+
+grapher_cycle_flags += --output-dir=$(CACHED_RPMS_DIR)/cache
+grapher_cycle_flags += --rpm-dir=$(RPMS_DIR)
+grapher_cycle_flags += --toolchain-rpms-dir=$(TOOLCHAIN_RPMS_DIR)
+grapher_cycle_flags += --toolchain-manifest=$(TOOLCHAIN_MANIFEST)
+grapher_cycle_flags += --tmp-dir=$(cache_working_dir)
+grapher_cycle_flags += --tdnf-worker=$(chroot_worker)
+grapher_cycle_flags += $(foreach repo, $(pkggen_local_repo) $(graphpkgfetcher_cloned_repo) $(REPO_LIST), --repo-file=$(repo))
+grapher_cycle_flags += $(grapher_extra_flags)
```

Comment:
For the sake of consistency, please include these flags in a way similar to what's happening in the `$(STATUS_FLAGS_DIR)/build-rpms.flag` target at the end of this file.

Reply:
Done

Reply:
What I had in mind, was to add the new flags directly inside the `$(graph_file)` target, right under `$(logging_command) \` (for instance). This way it's easier to see what is the input to the command and what files/directories it depends on.

For the optional flags added with the `ifeq ($(INPUT_ARGUMENT_NAME),y)`, I'm fine with either including them inline with this approach:

```
$(if $(filter y,$(INPUT_ARGUMENT_NAME)),--some-optional-flag) \
```

or by appending them to `grapher_extra_flags` as you do it now.